README.rst

```diff
@@ -1,7 +1,7 @@
 #####################
-ts_ATWhiteLightSource
+ts_atwhitelight
 #####################
 
-``ts_ATWhiteLightSource`` is a LSST Telescope & Site software package. Currently, it requires pymodbus as an external dependency. (https://github.com/riptideio/pymodbus)
+``ts_atwhitelight`` is a LSST Telescope & Site software package. Currently, it requires pymodbus as an external dependency. (https://github.com/riptideio/pymodbus)
 
 .. Add a brief (few sentence) description of what this package provides.
```

Comment:
Better add this now? Should be fairly simple..

Reply:
Absolutely. I rewrote the file.